### DISCUSSION TITLE
Updating Constraints in Local Space

### DISCUSSION THREAD

**Discussion by DennisSmolek** [jrouwe/JoltPhysics.js#135] [Q&A]
I have an object on a slider constraint that correctly moves and is limited along the axis:
![Screenshot 2024-03-29 205244](https://github.com/jrouwe/JoltPhysics.js/assets/1397052/036a08da-b9f5-4a2f-9025-0f05154845a1)

However, when I rotate the body it's attached to it of course does not follow it, and maintains the constraint along the worlds space axis. 

What I want however is to rotate the constraint along with the bodies:
![Screenshot 2024-03-29 205420](https://github.com/jrouwe/JoltPhysics.js/assets/1397052/44423508-59c1-4400-bbcf-f01d4f3f617c)

I tried setting the `mSpace` property to local, but this just makes the second body float aimlessly and I cant figure out what causes it to move when it's set this way. 

Is it possible to update the space that the consraint is operating in? or maybe I can update the constraint's axis at runtime?

**Comment by LeXXik** [comment]
The mSpace only affects the construction of the constraint. Once you created it, it should maintain the position relative to the connected body. It is hard to say from your screenshot what is happening or what those shapes are. Probably the other body connected to it is constraining its rotation and you don't actually rotate?

**Comment by DennisSmolek** [comment]
This was a bug on my part... 
I still have issues with SixDOF but I got this one with the Slider Resolved..
